app/helpers/books_helper.rb

```diff
@@ -0,0 +1,9 @@
+module BooksHelper
+  def check_img? book, file_img
+    book.image.attached? ? book.image : file_img
+  end
+
+  def same_au books
+    books.nil? ? (render "layouts/no_same_au") : (render books)
```

Comment:
`nil?` chị nghĩ chú nên dùng any? chứ 
vì `@books ` chị thấy nó là dạng collection nhé

Reply:
chị vẫn thấy lặp cái thèn `render` nên chú ... đưa cái render vào chỗ view là đẹp

Reply:
chỗ này thực ra chị chỉ muốn xử lý simple là `<%= render(@books) || "Book not same...." %>` là khỏe rồi cậu nà

Reply:
dạ vâng chị, e hiểu rồi